site/docs/patterns/preferences-dialog.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Preferences Dialog
+layout: DetailPattern
+aliases:
+  - /salt/patterns/preferences-dialog
+data:
+  resources:
+    [
+      { href: "https://www.saltdesignsystem.com/salt/components/navigation-item/examples", label: "Navigation item", internal: false },
+      { href: "https://www.figma.com/file/N6cSYycc4z6XkCjVvc1kIA/Preferences-Dialog-Pattern?type=design&node-id=157-40560&mode=design&t=eq3OapLjaquXm3UP-0", label: "Preferences Dialog stickhersheet", internal: true },
+    ]
+---
+
+A preferences dialog allows you to display, group and navigate application preferences within a responsive modal dialog window. This pattern introduces guidelines to make the preferences dialog consistent and usable.
+  
+  <Diagram
+    src="/img/patterns/preferences-dialog/preferences-dialog.png"
+    alt="Preferences Dialog example"
+    border={false}
+  />
+
+## When to use
+
+You should ideally house any editable settings within a centralized preferences dialog to display and manage user preferences across a platform, application or module. Displaying preferences should be contextual, i.e., the dialog should contain all preferences but depending on the entry point to the user interface, the preferences shown to you in the first instance will vary. 
+
+Furthermore, we recommend implementing consistent behavior around applying preferences, whether applying changes immediately or requiring the user to confirm any changes using the button bar actions. 
+
+## How to build
+
+### Anatomy
+
+A preferences dialog comprises four key features: 
+
+1. **Dialog:** a window with a content area that opens over the application content, focusing the user’s attention on a particular task or piece of information
```

Comment:
are we happy with calling it a window?

Reply:
We refer to dialog as a "window" in the component docs too... I would think "modal" is more fitting? Maybe something to check with Mike